src/main/webapp/WEB-INF/jsp/preview.jsp

```diff
@@ -246,7 +246,7 @@
                     <!-- Email preview table -->
                     <table id="email-list-table" class="list-group" role="list">
                         <tr rsf:id="row:" class="email-row list-group-item" role="listitem">
-                           <td class="right-content-email-toggle">
+                           <td class="right-content-email-toggle" role="presentation">
```

Comment:
Could the same look and behavior of table with aria roles

``` jsp
<table id="email-list-table" class="list-group" role="list">
    <tr rsf:id="row:" class="email-row list-group-item" role="listitem">
        <td class="right-content-email-toggle" role="presentation">
            ... (content here) ...
        </td>
    </tr>
</table>
```
 be achieved using list with an unordered list using `display: table-cell`?

``` jsp
<style>
    .table-cell {
        display: table-cell;
    }
</style>

<ul>
    <li rsf:id="row:" class="table-cell email-row">
        ... (content here) ...
    </li>
<ul>
```

The advantage of the change would be the DOM tree matching the accessibility tree, making maintenance easier.
The disadvantage of the change would be it uses CSS `display: table-cell;` which is considered bad practice.